process/interceptors/processor/miniblockInterceptorProcessor.go

```diff
@@ -71,11 +77,26 @@ func (mip *MiniblockInterceptorProcessor) Save(data process.InterceptedData, _ p
 		return err
 	}
 
+	isCrossShardForMe := mip.isMbCrossShardForMe(miniblock)
```

Comment:
Having a second thought, I think we should completely eliminate this check (isMbCrossShardForMe) because, otherwise, won't work with the hardfork mechanism.
If we reached this point is because either of these condition happened: 1. the miniblock was for this current shard, 2. the miniblock was already whitelisted or 3. both.

Reply:
ok